cns/service/main.go

```diff
@@ -1245,3 +1249,112 @@ func InitializeCRDState(ctx context.Context, httpRestService cns.HTTPService, cn
 
 	return nil
 }
+
+func SetMellanoxRegKey() {
```

Comment:
this should not be in main, it should be in a platform specific package

Reply:
Ok I will move this function to platform/os_windows.go ... make sense to me and thanks for example Ashvin.

Reply:
I will also follow the pattern of this function  SetSdnRemoteArpMacAddres and call it from main.. let me know if any concerns?